engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -516,14 +536,139 @@ public String spawnBlock(@Sender EntityRef sender, @CommandParam("blockName") St
         blockItem.send(new DropItemEvent(spawnPos));
         return "Spawned block.";
     }
+    @Command(shortDescription = "spawns many block together (value up to you prefer)", helpText = "you can select the particular block you want to spawn" +
```

Comment:
I'd prefer this description to be re-worded to fit better with the other commands. Eg, correct grammar (If English isn't your first language feel free to ask someone to check it before you make the PR), similar third person type description as the other commands. Eg, no "you"